pycuda_cudf_integration/README.md

```diff
@@ -0,0 +1,103 @@
+# Using PyCuda and Cudf
+## Overview
+Sometimes, a user may wish to run custom device code to do numerical operations on a `cuDF` column. 
+
+One way to do this is through custom cython bindings which can require significant setup work as shown [here](https://github.com/rapidsai/rapids-examples/tree/main/shareable-dataframes).
+
+With `PyCUDA`,  the custom CUDA kernels can be run directly using its` sourcModule` to modify the `cuDF`  dataframe.   
+
+`PyCUDA` has limitations, especially around running host-side code, which can are noted in detail in the limitations section.  
+
```

Comment:
Could use a brief section enumerating when to use PyCUDA and when to use an alternative (such as Cython/pybind11/etc.).